3p/integration.js

```diff
@@ -293,6 +294,7 @@ const AMP_EMBED_ALLOWED = {
   adsloom: true,
   adstyle: true,
   bringhub: true,
+  byplay: true,
```

Comment:
I don't think you need this unless you want users to use your network as `amp-embed`